site/docs/concepts/optimization-techniques/sparse-attention-prefill.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 2
+---
+
+# Sparse Attention prefill algorithms
+
+## Overview
+The sparse attention prefill algorithms enable speedups during the prompt processing (prefill) stage of the generation process by reducing the amount of computation taken up by the attention operation.
+During the prefill stage, the attention operation could only be applied to the subset of blocks determined to be most "important" for the current generation context, with importance estimation method determined by the algorithm.
+
+The KV-cache blocks that are deemed "unimportant" at a certain stage of prefill are not discarded entirely, but are preserved as-is so that they may still be considered for usage in the attention computation in latter stages of prompt processing.
+Moreover, the sparse prefill algorithms do not apply during the generation stage. 
+The sparse prefill algorithms therefore do not lead to decreased maximum and average memory consumption throughout the generation process, but they do lead to the decreased total generation and first token latency times due to enabling faster prefills.
+
+To achieve overall memory savings and generation stage memory and compute optimizations, the [cache eviction algorithm](./kvcache-eviction-algorithm.md) can be used along with sparse prefill algorithms, or separately.
+
+
+## Usage with openvino.genai
+The sparse attention prefill can be enabled by setting `.use_sparse_attention` field to `True` in the `openvino_genai.SchedulerConfig` structure that serves as input to most of the `openvino_genai` model pipeline objects.
+Further configuration of the sparse prefill algorithm is done using the `.sparse_attention_config` field of `openvino_genai.SchedulerConfig`, which accepts objects of `openvino_genai.SparseAttentionConfig`. See the [in-code documentation](../../../../src/cpp/include/openvino/genai/sparse_attention.hpp) for the description of individual fields; in particular, the `.mode` field of `SparseAttentionConfig` selects the type of the sparse prefill algorithm to be applied.
```

Comment:
That page is going to be deployed to https://openvinotoolkit.github.io/openvino.genai/docs/concepts/optimization-techniques/kvcache-eviction-algorithm. There are no sources. Add an absolute link. I think it should point to a specific commit instead of master so it would remain valid if the file is ever moved.

You can also deploy the docs with `Deploy Docs to GitHub Pages` action on your fork and a link to your PR description to verify everything is fine.